services/datamanager/builtin/sync_test.go

```diff
@@ -24,7 +24,7 @@ import (
 
 const (
 	syncIntervalMins = 0.0008
-	syncInterval     = time.Millisecond * 50
+	syncInterval     = time.Millisecond * 48 // Dan: 0.0008 * 60_000 = 48, not 50. Intentional choice?
```

Comment:
I'll let CI find out if this change is problematic.

Reply:
Not intentional, good update.